transcom/mymove
# [ MB-9033 ] Update node v16.15.0

## [Jira ticket](https://dp3.atlassian.net/browse/MB-9033) for this change

## Summary

Update Node to `16.15.0` from `14.18.1`. This work is being done in anticipation of updating _react-uswds_ to version `3.0.1`. That upgrade is being tracked in #8662 separately.

## Setup to Run Your Code

Load up the application locally and make sure that it builds. I way of doing this is by running `make clean build` which will clean out your local dependencies and then build the client and server. This should exit cleanly without errors. You may also fire up the application with `make client_run` and `make server_run` in separate terminals to ensure that the application is running. Lastly, run `make client_test` and `make e2e_test_fresh` to ensure the testing commands still run after this Node update.
## Verification Steps for Author

These are to be checked by the author.


See https://github.com/transcom/circleci-docker/pull/267 for updates to the underlying Docker image changes.
